middleware/cors/cors_test.go

```diff
@@ -196,7 +220,6 @@ func Test_CORS_Wildcard_AllowCredentials_Panic(t *testing.T) {
 		}()
 
 		app.Use(New(Config{
-			AllowOrigins:     "*",
```

Comment:
I know it's the default, but we should re-add this line given it's related to a CVE